storage/boltdb/client_test.go

```diff
@@ -78,3 +81,56 @@ func TestSuiteShared(t *testing.T) {
 		testsuite.RunTests(t, store)
 	}
 }
+
+type boltLongBenchmarkStore struct {
+	*Client
+	dirPath string
+}
+
+func (store *boltLongBenchmarkStore) BulkImport(iter storage.Iterator) (err error) {
+	// turn off syncing during import
+	oldval := store.db.NoSync
+	store.db.NoSync = true
+	defer func() { store.db.NoSync = oldval }()
+
+	var item storage.ListItem
+	for iter.Next(&item) {
+		if err := store.Put(item.Key, item.Value); err != nil {
+			return fmt.Errorf("Failed to insert data (%q, %q): %v", item.Key, item.Value, err)
+		}
+	}
+
+	return store.db.Sync()
+}
+
+func (store *boltLongBenchmarkStore) BulkDelete() error {
+	// do nothing here; everything will be cleaned up later after the test completes. it's not
+	// worth it to wait for BoltDB to remove every key, one by one, and we can't just
+	// os.RemoveAll() the whole test directory at this point because those files are still open
+	// and unremoveable on Windows.
+	return nil
+}
+
+func BenchmarkSuiteLong(b *testing.B) {
+	tempdir, err := ioutil.TempDir("", "storj-bolt")
```

Comment:
Since BulkDelete doesn't remove the folder anymore you would need here:
```
defer func(){
    err := os.RemoveAll(tempdir)
    if err != nil { b.Fatal(err) }
}()
```